cli/helm.go

```diff
@@ -15,32 +15,32 @@ import (
 // and should not be trusted.
 var version = "0.1.0"
 
-const globalUsage = `The Kubernetes package manager
+const globalUsage = `Helm Classic - The original Kubernetes package manager
```

Comment:
"The original Kubernetes package manager" sounds a little weird here, like it's claiming to be the first (i.e. 1000s of chains all claiming to be "The original pancake house"). Just a thought - not really a big deal IMO